spyder/app/tests/conftest.py

```diff
@@ -477,8 +477,8 @@ def main_window(request, tmpdir, qtbot):
                     for editorwindow in window.editor.editorwindows:
                         editorwindow.close()
                     editorstack = window.editor.get_current_editorstack()
-                    if editorstack.switcher_dlg:
-                        editorstack.switcher_dlg.close()
+                    if editorstack.switcher_plugin:
+                        editorstack.switcher_plugin.close()
```

Comment:
Depending on what is done to this method this could end up being removed or changed to something like:

    editorstack.switcher_plugin.on_close()

Or

    editorstack.switcher_plugin.get_container().on_close()